pkg/function/config.go

```diff
@@ -227,3 +229,39 @@ func parse(i *cue.Instance) (*Function, error) {
 
 	return f, nil
 }
+
+func formatCue(fn Function) (string, error) {
```

Comment:
Does this need to be capitalized to be exported?

Reply:
Added epxorted fns!